src/api/app/components/card_component.html.haml

```diff
@@ -1,9 +1,13 @@
 .card
-  .card-header.d-flex.justify-content-between
-    = header
+  .card-header
     - if delete_button?
-      .ms-4
-        = delete_button
+      .row
+        .col
+          = header
+        .col.col-auto
+          = delete_button
+    - else
+      = header
```

Comment:
Are you sure this does not break any other usage of the `CardComponent`?
I mean, what `header` can contain? it will not be wrapped by any of `.d-flex.justify-content-between` anymore: is this fine or?
I'm not saying it does break anything, but I would double check all the instances.

Reply:
> Are you sure this does not break any other usage of the `CardComponent`? I mean, what `header` can contain? it will not be wrapped by any of `.d-flex.justify-content-between` anymore: is this fine or? I'm not saying it does break anything, but I would double check all the instances.

You introduced `.d-flex.justify-content-between` in b2e4e22f65fb1c4fb113e6b5a938796eae451e0b a couple of years ago. These CSS classes improved the spacing of the elements in the card header. However, I don't think those classes were required back then, nor are they required now.

Reply:
Alright, thanks for double checking